contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Why your growth team isn't winning (and how to fix it)
+date: 2023-09-14
+author: ["lior-neu-ner"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/athlete-hog.jpeg
+featuredImageType: full
+category: Product growth
+tags: 
+  - Guides
+  - Product metrics
+  - Product engineers
+---
+
+There are tell-tale signs when a growth team isn't succeeding: Stuck metrics, missed KPIs, and burnt-out engineers. Everyone is frustrated, but how do you fix it?
+
+[Alexey Komissarouk](https://alexeymk.com/), former head of growth engineering at [MasterClass](https://www.masterclass.com/), has seen it all. He grew Masterclass xyz... 
+
+In this post, Alexey shares his advice on how to get your growth team winning again.
+
+TLDR; there are 3 areas to investigate:
```

Comment:
I'm not sure if this is exactly the right hook. I think you are considering between two hooks here:

1. Sometimes a growth team is in a rut and here is how you fix it (in which case the first topic is more relevant than the last two)
2. Performance Tuning Growth Team - IE, the framework for thinking about improving your team's performance and ideas for improving each part.

I think 2 is more interesting philosophically, but I get why 1 might be a better hook - but then I would cut some of the non-engineering related parts in the name of keeping it tight. I rambled a bit on our call but that doesn't mean we have to include all of it 😄

Reply:
I've removed this section now